### DISCUSSION TITLE
[Feture Request] Touchpad remmaping support?

### DISCUSSION THREAD

**Discussion by wzhchin** [xremap/xremap#606] [Ideas]
Would you please consider touchpad support?

Like multi-finger gestures and edge scrolling?

**Comment by k0kubun** [comment]
I don't have a touchpad, so I need somebody else to implement it and be a maintainer of it.

Also, how would gestures and edge scrolling fit into xremap's config? Can you share an example config in YAML you'd like to use?

**Comment by wzhchin** [comment]
Thanks for your reply, I have no idea about it now, but I would like to implement it, maybe I will show you them at the time.

Future more, do you have any idea to use xremap to insert unicode chars(I'm using hyprland now)?

**Comment by k0kubun** [comment]
xremap is implemented over evdev, and there's no concept of unicode at such a lower layer. If you want to input unicode, first you need to be able to do that without xremap, and then map something to that action with xremap. If you have a CLI to do that, for example, you could run that with the "launch" syntax. If you need to input some special sequence, xremap could help you input that with different keys.
